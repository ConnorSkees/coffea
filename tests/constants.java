public class constants {

    public static void main(String[] args) {
        int i1 = 0;
        int i2 = 1;
        int i3 = 2;
        int i4 = 3;
        int i5 = 4;
        int i6 = 5;

        float f1 = 0.0f;
        float f2 = 1.0f;
        float f3 = 2.0f;

        double d1 = 0.0d;        
        double d2 = 1.0d;        

        long l1 = 0l;
        long l2 = 1l;

        double largeDouble = Double.MAX_VALUE;
        long largeLong = Long.MAX_VALUE;

        char c1 = 4;
        byte b1 = 4;
        short s1 = 4;
        boolean bool0 = false;
        boolean bool1 = true;
    }
}