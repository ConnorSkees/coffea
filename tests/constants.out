public class constants {
  public static void main(String[] arg0) {
    int i1 = 0;
    int i2 = 1;
    int i3 = 2;
    int i4 = 3;
    int i5 = 4;
    int i6 = 5;
    float f7 = 0;
    float f8 = 1;
    float f9 = 2;
    double d10 = 0;
    double d12 = 1;
    long l14 = 0;
    long l16 = 1;
    double d18 = 179769313486231570000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
    long l20 = 9223372036854775807;
    int i22 = 4;
    int i23 = 4;
    int i24 = 4;
    int i25 = 0;
    int i26 = 1;
    return;
  }
}
